src/lib/registration/eula_reader.rb

```diff
@@ -33,6 +33,8 @@ class EulaReader
 
```

Comment:
drop the angle brackets in "{ \<locale> => \<file_name> }" otherwise HTML shows only { =>  }